bucket/psreadline.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "2.2.2",
+    "description": "A bash inspired readline implementation for PowerShell",
+    "homepage": "https://github.com/PowerShell/PSReadLine",
+    "license": "BSD-2-Clause",
+    "notes": [
+        "Use the module by running: 'Import-Module PSReadLine'",
```

Comment:
AFAIK, the highest version of PSReadline is imported automatically, if found in PSModulePath. So this might not be needed.

Reply:
You are right - just validated and it works without explicit import. Fixed.